lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -1098,94 +1100,6 @@ export class BrowserCacheManager extends CacheManager {
         }
     }
 
-    /**
-     * Gets a list of accounts that match all of the filters provided
-     * @param account
-     */
-    getAccountInfoByFilter(
-        accountFilter: Partial<Omit<AccountInfo, "idTokenClaims" | "name">>
-    ): AccountInfo[] {
-        const allAccounts = this.getAllAccounts();
-        this.logger.trace(
-            `BrowserCacheManager.getAccountInfoByFilter: total ${allAccounts.length} accounts found`
-        );
-
-        return allAccounts.filter((accountObj) => {
-            if (
-                accountFilter.username &&
-                accountFilter.username.toLowerCase() !==
-                    accountObj.username.toLowerCase()
-            ) {
-                return false;
-            }
-
-            if (
-                accountFilter.homeAccountId &&
-                accountFilter.homeAccountId !== accountObj.homeAccountId
-            ) {
-                return false;
-            }
-
-            if (
-                accountFilter.localAccountId &&
-                accountFilter.localAccountId !== accountObj.localAccountId
-            ) {
-                return false;
-            }
-
-            if (
-                accountFilter.tenantId &&
-                accountFilter.tenantId !== accountObj.tenantId
-            ) {
-                return false;
-            }
-
-            if (
-                accountFilter.environment &&
-                accountFilter.environment !== accountObj.environment
-            ) {
-                return false;
-            }
-
-            return true;
-        });
-    }
-
-    /**
-     * Checks the cache for accounts matching loginHint or SID
-     * @param loginHint
-     * @param sid
-     */
-    getAccountInfoByHints(
```

Comment:
This is already supported by `getAccountInfoFilteredBy` in the parent CacheManager

Reply:
Note: Only the sid match was missing and is implemented in this PR